internal/api/v1/router.go

```diff
@@ -17,6 +17,12 @@ func Router() *httprouter.Router {
 	router.HandlerFunc("GET", "/api/v1/orgs", OrganizationsController{}.Index)
 	router.HandlerFunc("POST", "/api/v1/orgs", OrganizationsController{}.Create)
 
+	router.HandlerFunc("GET", "/api/v1/orgs/:org/services", ServicesController{}.Index)
+	router.HandlerFunc("GET", "/api/v1/orgs/:org/services/:service", ServicesController{}.Show)
+
+	router.HandlerFunc("GET", "/api/v1/orgs/:org/serviceclasses", ServiceClassesController{}.Index)
+	router.HandlerFunc("GET", "/api/v1/orgs/:org/serviceclasses/:serviceclass/serviceplans", ServicePlansController{}.Index)
```

Comment:
Neither service classes nor plans are linked to orgs in any way.
IMHO the routes should be `/api/v1/serviceclasses` and `/api/v1/serviceclasses/:serviceclass/serviceplans`.

Reply:
Done.